media/js/foundation/annual2011.js

```diff
@@ -215,12 +209,10 @@
 
     // Set up the contributor stories carousel
     function setupCarousel() {
-        $sliderPrime.clone().attr("id", "story-slider-clone").insertAfter($("#video-stage"));
-        if ( $("#story-vid").length > 0 ) {
-            $("#story-vid")[0].pause();
-        }
-        $sliderPrime.hide();
-        $("#story-slider-clone").jcarousel({
+        $('.jcarousel-clip').jcarousel().on('jcarousel:destroyend', function() {
+            $sliderPrime.attr('style', '');
```

Comment:
Ah, I think this isn't removing the styles because the `.jcarousel()` call immediately following overwrites these settings. The way to make it work would be adding the `.on('jcoursel:destroyend', ...` bit on to the initialization call below. But let's not do that!

I'd say you can wipe out this bit and we'll just use your CSS solution.